Felienne/hedy
# [IMPROVEMENT] Only re-loads profile page on language change

**Description**
In this PR we improve the profile change implementation so we only re-load the page when necessary; when changing the profile language. In any other situation we don't have to re-load the page and can simply return the user some feedback. We also switched from the form-styled feedback box to the `modal.alert()` feedback to return the user information.

**Fixes #1955**

**How to test**
Update your profile with and without setting a new profile language. Notice that in both cases the modal.alert() is returned with feedback instead of the currently used feedback box below the form. Verify that in both cases a different message is returned and when changing the language the page is re-loaded.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
